test/testhelpers.sh

```diff
@@ -148,3 +148,9 @@ shutdown() {
     fi
   fi
 }
+
+assert_file_line_count() {
+  local filename="$1"
+  local lines="$2"
+  [ $(wc -l < "$filename") -eq "$lines" ]
```

Comment:
"easier to read and use" really depends on the context. In this case I'm preferring short shell commands to extra overhead from helpers. Thanks for reverting.